packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -413,16 +601,22 @@ export default class StatsAnalyzer extends EventsScope {
         LoggerProxy.logger.info('StatsAnalyzer:index#processInboundRTPResult --> Packets getting lost from the receiver ', this.statsResults[mediaType][sendrecvType].currentPacketLossRatio);
       }
 
-      this.statsResults[mediaType][sendrecvType].totalPacketsLost = result.packetsLost;
+      // TODO: check the packet loss value is negative values here
+
+      if (result.packetsLost) {
+        this.statsResults[mediaType][sendrecvType].totalPacketsLost = result.packetsLost > 0 ? result.packetsLost : -result.packetsLost;
```

Comment:
We are just assuming a negative packet loss needs to be switched to positive?

Reply:
I have seen in chrome the packet loss was negative values, So need to change that

Reply:
Should we change a negative value to 0 instead? Changing a `-100` to `100` seems like a bad solution unless we know for a fact it is a browser bug that is reporting the value incorrectly.

Reply:
I was seeing packet loss as -100 meaning around 100 packets were lost